### PR TITLE
Added reviewer's name ro reviews given to restaurants

### PR DESCRIPTION
**Pull request by hanskw4267**

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -171,6 +171,7 @@ create table Orders
 create table Reviews
 (
     oid         integer primary key not null references Orders (id) on delete cascade,
+    cid         varchar(20) references Customers (id),
```
Comment (liujiajun) [inline database/schema.sql:174]: This is not necessary since each cid is uniquely linked to one cid

**Review by liujiajun — CHANGES_REQUESTED** [review]
This is not necessary since each cid is uniquely linked to one cid

File: database/schema.sql (hunk 1 above)
Comment (hanskw4267) [reply database/schema.sql:174]: Did it this way so that easier to retrieve cid for each review when we display. Rather than using query for each review.

Comment (liujiajun) [reply database/schema.sql:174]: Unfortunately this creates redundant attributes in the schema, which I think should be avoided in the context of 2102 unless there are strong justifications (e.g. frequent read access)

Comment (hanskw4267) [reply database/schema.sql:174]: Alright. Got ur point. Will do

**Pull request closed without merging**